doc/nrf/gs_programming.rst

```diff
@@ -178,31 +178,77 @@ For more information on building and programming using the command line, see the
+| Hardware platform | PCA number | Board name                                                      | Build target                          |
++===================+============+=================================================================+=======================================+
+| nRF52 DK          | PCA10040   | :ref:`nrf52dk_nrf52832 <zephyr:nrf52dk_nrf52832>`               | ``nrf52dk_nrf52832``                  |
+| (nRF52832)        |            +-----------------------------------------------------------------+---------------------------------------+
+|                   |            | :ref:`nrf52dk_nrf52810 <zephyr:nrf52dk_nrf52810>`               | ``nrf52dk_nrf52810``                  |
++-------------------+------------+-----------------------------------------------------------------+---------------------------------------+
+| nRF52833 DK       | PCA10100   | :ref:`nrf52833dk_nrf52833 <zephyr:nrf52833dk_nrf52833>`         | ``nrf52833dk_nrf52833``               |
+|                   |            +-----------------------------------------------------------------+---------------------------------------+
+|                   |            | :ref:`nrf52833dk_nrf52820 <zephyr:nrf52833dk_nrf52820>`         | ``nrf52833dk_nrf52820``               |
++-------------------+------------+-----------------------------------------------------------------+---------------------------------------+
+| nRF52840 DK       | PCA10056   | :ref:`nrf52840dk_nrf52840 <zephyr:nrf52840dk_nrf52840>`         | ``nrf52840dk_nrf52840``               |
+|                   |            +-----------------------------------------------------------------+---------------------------------------+
+|                   |            | :ref:`nrf52840dk_nrf52811 <zephyr:nrf52840dk_nrf52811>`         | ``nrf52840dk_nrf52811``               |
++-------------------+------------+-----------------------------------------------------------------+---------------------------------------+
+| nRF52840 Dongle   | PCA10059   | :ref:`nrf52840dongle_nrf52840 <zephyr:nrf52840dongle_nrf52840>` | ``nrf52840dongle_nrf52840``           |
++-------------------+------------+-----------------------------------------------------------------+---------------------------------------+
+| Thingy:52         | PCA20020   | :ref:`thingy52_nrf52832 <zephyr:thingy52_nrf52832>`             | ``thingy52_nrf52832``                 |
++-------------------+------------+-----------------------------------------------------------------+---------------------------------------+
+| nRF5340 PDK       | PCA10095   | :ref:`nrf5340pdk_nrf5340 <zephyr:nrf5340pdk_nrf5340>`           | ``nrf5340pdk_nrf5340_cpunet``         |
+|                   |            |                                                                 |                                       |
+|                   |            |                                                                 | ``nrf5340pdk_nrf5340_cpunetns``       |
```

Comment:
There is no such target. It should be `nrf5340pdk_nrf5340_cpuappns`.

Reply:
Fixed, thank you